src/config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
     ignorePackages: [],
     packagerOptions: {},
     tsConfig: "tsconfig.json",
+    forktschecker: false,
```

Comment:
I wasn't sure if ForkTsChecker plugin should be applied by default or not, so for now I have disabled it, let me know if this need to be reverted